Sources/BraintreePayPal/BTPayPalCheckoutRequest.m

```diff
@@ -0,0 +1,55 @@
+#import "BTPayPalCheckoutRequest_Internal.h"
+#import "BTPayPalRequest_Internal.h"
+
+@implementation BTPayPalCheckoutRequest
+
+- (instancetype)initWithAmount:(NSString *)amount {
+    if (amount == nil) {
+        return nil;
+    }
+
+    if (self = [super init]) {
+        _amount = amount;
+        _offerPayLater = NO;
+        _intent = BTPayPalRequestIntentAuthorize;
+    }
+    return self;
+}
+
+- (NSDictionary<NSString *, NSObject *> *)parametersWithConfiguration:(BTConfiguration *)configuration isBillingAgreement:(BOOL)isBillingAgreement {
```

Comment:
The `isBillingAgreement` param will go away once we add the `BTPayPalVaultRequest` subclass.